app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -290,7 +290,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       ($scope.currentTab == "service_account" && $scope.emsCommonModel.service_account != ''))) {
       return true;
     } else {
-      return false;
+      return true;
```

Comment:
@jameswnl Not sure if I understand this change.
The BZ is specifically mentioning the RHV provider, so can you provide more info on how this change is related to the issue described in the BZ?